src/components/TwoColumnList/TwoColumnList.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { forwardRef, ReactElement } from 'react';
+import cx from 'classnames';
+import { useThemeContext } from '~/contexts';
+import { Heading } from '~/components/Heading';
+import { Hyperlink } from '~/components/Hyperlink';
+import { List } from '~/components/List';
+import type { ListItem } from '~/components/List/List.types';
+import type { TwoColumnListProps, ColumnProps } from './TwoColumnList.types';
+import styles from './TwoColumnList.module.css';
+
+const TwoColumnList = forwardRef<any, TwoColumnListProps>(
```

Comment:
The ref is assigned to a div, would `HTMLDivElement` be a suitable type to use instead of any?
```suggestion
const TwoColumnList = forwardRef<HTMLDivElement, TwoColumnListProps>(
```

Reply:
Very true. context switching from the other PR that was similar must have confused me. thank you